### PR TITLE
Add link to Find a Club on homepage

### PR DESCRIPTION
**Pull request by zanedb**

### CODE REVIEW

**Comment by lachlanjc** [comment]
Cool, can you please provide screenshots of this on desktop and mobile?

**Comment by zanedb** [comment]
Yeah, have to build first.

**Comment by zanedb** [comment]
I think it looks pretty good on desktop:
desktop

Mobile is a bit iffy (but already sort of was):
mobile

**Comment by lachlanjc** [comment]
Yeah. I totally support adding a link to Finder but I think we need a new navigation system. Already “Our Team” is totally unconnected from the Workshops shortcut. I’m not sure what the best solution is here, but in the short term, maybe switch the Donate button (in the Bubbles component) to “Find a Club”, and add Donate in the middle of the nav bar?

**Comment by zanedb** [comment]
Works for me. I'll open an issue and change that.

**Comment by zanedb** [comment]
Fixed. Look good? @lachlanjc

File: src/components/Bubbles.js [hunk 1]
```diff
@@ -126,8 +126,8 @@ const Bubbles = ({ children }) => (
         things apart.
       </Text>
       <Flex justify="center" wrap mx={-2} mt={4}>
-        <LargeButton href="/donate" inverted m={2}>
-          Donate
+        <LargeButton href="https://finder.hackclub.com" inverted m={2}>
+          Find Clubs
```
Comment (lachlanjc) [inline src/components/Bubbles.js]: I think “Find a Club” would be better, only if you’re in a very dense area are there many nearby

**Comment by lachlanjc** [comment]
Yeah, I think this flow is better. Does the “Donate” link still put “Our Team” in two rows on mobile?

File: src/components/Bubbles.js (hunk 1 above)
Comment (zanedb) [reply src/components/Bubbles.js]: Yup, that's what I had at first. Only changed it to make the text smaller, as "Donate" is less characters than "Find a Club." Should I change it back? @lachlanjc

**Comment by zanedb** [comment]
Checking/building right now.

**Comment by zanedb** [comment]
And.. honestly it looks a bit weirder. But I do like having the finder be a large button much better.

mobile2

**Pull request merged**